src/main/java/fi/metatavu/metamind/liquibase/LiquibaseProducer.java

```diff
@@ -0,0 +1,46 @@
+package fi.metatavu.metamind.liquibase;
+
+import java.sql.SQLException;
+
+import javax.annotation.Resource;
+import javax.enterprise.context.Dependent;
+import javax.enterprise.inject.Produces;
+import javax.sql.DataSource;
+
+import liquibase.integration.cdi.CDILiquibaseConfig;
+import liquibase.integration.cdi.annotations.LiquibaseType;
+import liquibase.resource.ClassLoaderResourceAccessor;
+import liquibase.resource.ResourceAccessor;
+
+/**
+ * Liquibase producer
+ * 
+ * @author Antti Leppä
+ */
+@Dependent
+public class LiquibaseProducer {
+  
+  @Resource (lookup = "java:jboss/datasources/metamind-api")
+  private DataSource dataSource;
+  
+  @Produces
+  @LiquibaseType
+  public CDILiquibaseConfig createConfig() {
+    CDILiquibaseConfig config = new CDILiquibaseConfig();
+    config.setChangeLog("fi/metatavu/metamind/changelog.xml");
+    return config;
+  }
+  
+  @Produces
+  @LiquibaseType
+  public DataSource createDataSource() throws SQLException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.sql.SQLException', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)